### DISCUSSION TITLE
[help]Force

### DISCUSSION THREAD

**Discussion by wu-dennis** [Chia-Network/chia-blockchain#3165] [General]
5TB HDD  can create 45 polts,how much is it to  dig out?
Total Network Space 1.061EIB

**Comment by sargonas** [comment]
Tagging this issue for migration to a discussion. Apologies in advance, but you will be best served by moving this over there and out of the issue tracker, where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

^MigrateToDiscussion
